src/Sep.Test/SepReaderColsTest.cs

```diff
@@ -28,11 +28,39 @@ public void SepReaderColsTest_Length()
         Run(cols => Assert.AreEqual(_colsCount, cols.Length));
     }
 
+    [TestMethod]
+    public void SepReaderColsTest_ToStringsArray()
+    {
+        Run(cols => CollectionAssert.AreEqual(_colTexts, cols.ToStringsArray()));
+    }
+
+    [TestMethod]
+    public void SepReaderColsTest_ToStrings()
+    {
+        Run(cols => CollectionAssert.AreEqual(_colTexts, cols.ToStrings().ToArray()));
+    }
+
+    [TestMethod]
+    public void SepReaderColsTest_ParseToArray()
+    {
+        Run(cols => CollectionAssert.AreEqual(_colValues, cols.ParseToArray<int>()));
+        Run(cols => CollectionAssert.AreEqual(_colValuesFloat, cols.ParseToArray<float>()));
+        Run(cols => CollectionAssert.AreEqual(_colTexts, cols.ToStringsArray()));
```

Comment:
@nietras Overall `ToStringsArray` is nice to have, but it might also be a source of confusion since the same functionality is provided in two different ways; `ToStringsArray` vs. `ParseToArray<string>` -  at least from .NET 8 and forward. For that reason, I would consider leaving out `ToStringsArray`. Seems more clean to me just having one way of parsing to array.

Reply:
My concern is this leaves .NET 7 users in the dust, so I think it is fine to have the two ways. It's not like ParseToArray is redundant or useless.